File: context-kit/index-tests/corpus/sample.rs
pub mod things;

pub struct Alpha;

pub enum Beta {
    A,
    B,
}

pub trait Gamma {
    fn g(&self);
}

pub fn delta() {}

pub(crate) fn theta() {}

pub const EPSILON: u32 = 3;

pub static ZETA: u32 = 4;

pub type Eta = u32;

fn private_not_shown() {}

File: context-kit/index-tests/corpus/sample.ts
export function alpha() {}

export async function beta(): Promise<void> {}

export class Gamma<T> {}

export interface Delta {
    x: number;
}

export type Epsilon = number;

export enum Zeta {
    A,
    B,
}

export const enum Eta {
    X,
}

export const theta = 1;

export let iota = 2;

export var kappa = 3;

export default function main() {}

export { alpha as renamedAlpha } from "./other";
export * from "./more";

const notExported = 4;
function alsoPrivate() {}
